task/common/values.go

```diff
@@ -41,7 +41,15 @@ type Event struct {
 	Code        string
 	Description []string
 }
-type Storage struct{}
+type RemoteStorage struct {
```

Comment:
Is the `Remote` prefix needed? 💅🏼

Reply:
I think it makes it more explicit that we're dealing with cloud storage.